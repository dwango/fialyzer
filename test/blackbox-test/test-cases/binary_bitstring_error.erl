-module(binary_bitstring_error).
-export([main/1]).

main(X) ->
    _ = binary_to_integer(X), %% binary -> integer.
    _ = bit_size(X). %% bitstring -> integer >= 0.
