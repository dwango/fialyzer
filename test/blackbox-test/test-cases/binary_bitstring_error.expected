TODO:filename:5: Type error: type mismatch;
  found   : <<_:0, _:_*1>>
  required: <<_:0, _:_*8>>
there is no solution that satisfies subtype constraints
